### PR TITLE
Initial .NET Playground

### PR DESCRIPTION
**Pull request by TorstenDittmann**
- Update Readme
- Flow:
  - Creates a user
  - Lists all users
  - Creates a Collection
  - List Collections
  - Creates Documents
  - List all Documents

### CODE REVIEW

**Comment by TorstenDittmann** [comment]
@eldadfux can't assign for review, so just gonna tag you.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,43 @@
...
+
+We truly ❤️ pull requests! If you wish to help, you can learn more about how you can contribute to this project in the [contribution guide](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md).
+
+## Security
+
+For security issues, kindly email us [security@appwrite.io](mailto:security@appwrite.io) instead of posting a public issue in GitHub.
+
+## Follow Us
+
+Join our growing community around the world! Follow us on [Twitter](https://twitter.com/appwrite_io), [Facebook Page](https://www.facebook.com/appwrite.io), [Facebook Group](https://www.facebook.com/groups/appwrite.developers/) or join our live [Gitter community](https://gitter.im/appwrite/community) for more help, ideas and discussions.
```
Comment (eldadfux) [inline README.md]: We should probably replace Gitter with Discord (https://appwrite.io/discord), which repo you used as template? we should probably fix it there too.

**Review by eldadfux — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (TorstenDittmann) [reply README.md]: Copy & pasted it from https://github.com/appwrite/playground-for-web 👍

Comment (TorstenDittmann) [reply README.md]: Opened https://github.com/appwrite/playground-for-web/pull/20 👍

**Review by eldadfux — APPROVED** [review]

**Comment by eldadfux** [comment]
🥳

**Pull request merged**